lxc/config/config.go

```diff
@@ -36,6 +36,14 @@ type Config struct {
 	cookieJars map[string]*cookiejar.Jar
 }
 
+// GlobalConfigPath returns a joined path of the global configuration directory and passed arguments
+func (c *Config) GlobalConfigPath(paths ...string) string {
+	path := []string{"/etc/lxd"}
```

Comment:
Can you make this consider `LXD_GLOBAL_CONF` environment and use that over `/etc/lxd/` if set?

That should line up with what we do for `LXD_CONF` and will be needed for the snap package.

Reply:
@stgraber is it ok? Or should we add new `GlobalConfigDir` field to [Config](https://github.com/lxc/lxd/blob/master/lxc/config/config.go#L11)?